datasette_write_ui/static/table.css

```diff
@@ -63,7 +63,7 @@ td.type-pk {
   left: 0;
 }
 
-.row-icon .menu {
+.datasette-write-ui-menu {
```

Comment:
I think we may want to set a convention more like this in general for plugins - I worry about plugins adding their own styles that later conflict with other plugins, or changes to Datasette Core, or changes that a user makes in their own custom stylesheet.